src/components/data_comps/datm/datm_comp_mod.F90

```diff
@@ -387,8 +385,6 @@ subroutine datm_comp_init(Eclock, x2a, a2x, &
        kswvdr= mct_aVect_indexRA(a2x,'Faxa_swvdr')
        kswvdf= mct_aVect_indexRA(a2x,'Faxa_swvdf')
        kswnet= mct_aVect_indexRA(a2x,'Faxa_swnet')
-       kco2p = mct_aVect_indexRA(a2x,'Sa_co2prog',perrWith='quiet')
-       kco2d = mct_aVect_indexRA(a2x,'Sa_co2diag',perrWith='quiet')
```

Comment:
Just want to confirm -- looks like these two lines are defunct code that you are cleaning up and removing them will not affect outputs to the coupler... is that correct?

Reply:
That is correct. These were not used anywhere so it was confusing to set them.

Reply:
great, thanks for the response!